packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -48,6 +51,18 @@ contract SystemConfig is OwnableUpgradeable, ISemver, IGasToken {
         address gasPayingToken;
     }
 
+    /// @notice Struct representing the roles of the system.
+    /// @notice The owner (chain operator) of the system.
+    /// @notice The fee admin of the system.
+    /// @notice The unsafe block signer of the system.
+    /// @notice The batcher hash of the system.
+    struct Roles {
+        address owner;
+        address feeAdmin;
+        address unsafeBlockSigner;
+        bytes32 batcherHash;
```

Comment:
I would definitely consider just turning the batcher hash into an address for simplicity and always assume version 0 until we explicitly want to move away from version 0 but no need for additional churn to this PR at this point

Reply:
I'd prefer not to. It will probably be a large diff to change the type to address, and then the name `batcherHash` will become even more confusing unless we also change it.

Reply:
Resolving as the latest round of review did not mention this.